taxonomy/models.py

```diff
@@ -260,3 +260,78 @@ def __repr__(self):
         return '<Skill id="{0}" name="{1}" job="{2!r}">'.format(
             self.id, self.name, self.job,
         )
+
+
+class JobPostings(TimeStampedModel):
+    """
+    Job postings for a job.
+
+    .. no_pii:
+    """
+
+    job = models.ForeignKey(
+        Job,
+        blank=False,
+        null=False,
+        on_delete=models.deletion.CASCADE,
+        help_text=_(
+            'The ID of the job to filter job postings by.'
+        )
+    )
+
+    median_salary = models.DecimalField(
+        blank=False,
+        null=False,
+        decimal_places=2,
+        max_digits=12,
+        help_text=_(
+            'The median annual salary advertised on job postings for the job.'
+        )
+    )
+
+    median_posting_duration = models.FloatField(
```

Comment:
These all results are coming in decimal form? As we are storing them in FloatFields.

Reply:
I have converted all of them to DecimalField and have updated the migration as well.